{{cookiecutter.project_slug}}/clients/vue3/src/views/PageNotFound.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div class="page-not-found">
+    <h1 class="page-not-found__error">404</h1>
+    <img alt="Project Logo" src="../assets/logo.png" />
```

Comment:
**Question:** is it worth adding the logo here? I think it adds a little more to the page, but then folks will have to remember to replace the logo image with the [client] project specific one and not sure if then it'd be easier to just not have an image here.

If we did want to keep the logo here, however, we could always add a reminder/note to the Vue `README.md` file.

Reply:
This page existing is good enough for now.
Eventually all pages should be at least half-decently designed with things like logos in a good location.
There was a block of comments in the README for how to swap out the favicons and logos. That got removed but will get added back soon. Thats another step devs should be doing shortly after bootstrapping.

sooo...perhaps having the logo here is good as it reminds the devs they still need to do that step?